### PR TITLE
[Overflow-14] Implement API to Sidebars

### PR DESCRIPTION
**Pull request by IanUrriza10**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-14
## Expected Output
Dynamic Watched Tag List based on user_id
Dynamic Teams List based on user_id
Working Add/Remove Watched Tags
Dynamic Teams List Based on team_id

## Screenshots
image
image
image

### CODE REVIEW

File: frontend/components/templates/layouts/index.tsx [hunk 1]
```diff
@@ -33,6 +33,16 @@ const Layout = ({ children }: LayoutProps) => {
...
+                 <ToastContainer
+                autoClose={5000}
+                hideProgressBar={false}
+                newestOnTop={false}
+                closeOnClick
+                rtl={false}
+                pauseOnFocusLoss
+                draggable
+                pauseOnHover
+            />
```
Comment (nicoleamber) [inline frontend/components/templates/layouts/index.tsx:45]: Format properly

File: frontend/components/templates/layouts/index.tsx [hunk 2]
```diff
@@ -33,6 +33,16 @@ const Layout = ({ children }: LayoutProps) => {
                         <Image src="/images/sun_logo.png" alt="sun-logo" fill />
                     </div>
                 )}
+                 <ToastContainer
```
Comment (nicoleamber) [inline frontend/components/templates/layouts/index.tsx:36]: Format properyl

File: frontend/components/templates/layouts/index.tsx [hunk 3]
```diff
@@ -33,6 +33,16 @@ const Layout = ({ children }: LayoutProps) => {
...
+                autoClose={5000}
+                hideProgressBar={false}
```
Comment (nicoleamber) [inline frontend/components/templates/layouts/index.tsx]: Format properly

**Review by jules-lucero — CHANGES_REQUESTED** [review]
Please run the Prettier script to apply proper formatting

**Review by nicoleamber — APPROVED** [review]

**Review by jules-lucero — APPROVED** [review]

**Comment by ipilikent** [comment]
### Description
- The role is missing from the Member list on the Team Page

Screenshot:



### Expected Behavior
- The Member list must have a role on the Team Page

**Comment by ipilikent** [comment]
## Proof
- User can Add tags and Remove tags
- Dynamic Watched Tag List and Dynamic Teams List is displayed
- The error and success messages are displayed

<details>

14 - FE Sidebar Questions

14 - FE Sidebar Teams

14 - FE Sidebar Questions

</details>

Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=1059275291 
-----
- Total Test time: 1 hr
- Total Comments: 1
- Total Iterations: 1

**Pull request merged**